kernel/include/paging.h

```diff
@@ -0,0 +1,53 @@
+//https://wiki.osdev.org/Paging         x86 2 layer paging
+//https://os.phil-opp.com/page-tables/  x64 4 layer paging
+//P4 P3 P2 P1 represent the different pagetable layers
+
+
+//uefi pagetables are placed in a nonreadable section by default
+
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+//temporary solution before memory allocation is done
+extern uint64_t page_zone_stack[] __attribute__((aligned(4096)));
```

Comment:
The attribute should be placed on the variable in the source file and not the header.